templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,11 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 31**
```

Comment:
I put this in Feature level 30 since we hadn't actually bumped API_FEATURE_LEVEL yet.